### ISSUE TITLE
Requested symbols

### ISSUE THREAD

**Issue by SteveALee** [mulberrysymbols/mulberry-symbols#58]
Pronouns

- You
- We
- He
- She
- They, etc

**Comment by eadraffan** [comment]
This is hopefully something we can help with if we get our Symbol Creator working again, but I am not sure whether this works for you?  I think it is important to keep the style for some AAC users - by that I mean the look and feel!   
![you](https://user-images.githubusercontent.com/743388/85588043-3adad600-b63a-11ea-812f-5535e926803b.png)

**Comment by SteveALee** [comment]
Thanks @eadraffan . Yes I think it works, but maybe an improvement would be to somehome highlight the hand is another persons so it's not "me" or "I". Then move the hand to look like I?

I do hope the symbol editor can be got working again as compound Symbols could be a new source of symbols. We'd need SVG source though.

**Comment by eadraffan** [comment]
Yes they do need to be svg and I agree with all you say about angles and other skin tones etc!

**Comment by eadraffan** [comment]
Here are some missing prepositions (some used as adverbs) and conjunctions.  Options from ARSAAC and Bliss given as examples in places. 
[Mulberry Symbols missing Prepositions23032021.docx](https://github.com/mulberrysymbols/mulberry-symbols/files/6188588/Mulberry.Symbols.missing.Prepositions23032021.docx)

**Comment by eadraffan** [comment]
Here are the Personal Pronouns for English with an example from ARASAAC.  I have also added a table for French and Arabic pronouns, so we remember differences in other languages.  Questions to be answered - do we use a face that can be used for male and female where relevant?  We know stick figures do not suit everyone and are not part of the Mulberrry set.  What thickness of outline do we use for the symbols as these vary on the set.  Do we use just a head or whole body for all the pronoun symbols?  I have made an example of a man and a woman in the attached document.  (outline 5.25pt)

**Comment by eadraffan** [comment]
Here is a Word doc with some sample gaps in the verbs and many are uncategorised and best viewed on the web page of categories. These are complex symbols, but there are some good examples of bodies with blank faces carrying out the actions - do these allow us to avoid having to think about gender options? 
 https://github.com/mulberrysymbols/mulberry-symbols/releases/latest/download/categories.html
[Mulberry Symbols Verbs23032021.docx](https://github.com/mulberrysymbols/mulberry-symbols/files/6189179/Mulberry.Symbols.Verbs23032021.docx)
